### PR TITLE
Added test cases

### PR DESCRIPTION
**Pull request by kamransagheir**
pleas review the test cases.

### CODE REVIEW

**Comment by Hussainalbaidhani** [comment]
reviewed.

File: src/test/java/Bookstore/Model/BookTest.java [hunk 1]
```diff
@@ -25,34 +19,53 @@
...
+	@Test
+	void testGetBookName() {
+		assertEquals("testBook", book.getBookName());
 	}
 
 	@Test
-	void testGetOwner() {
-		assertEquals( "testOwner", (bookStore.getOwner()).getName());
+	void testGetPicture() {
+		assertEquals("test book", book.getDescription());
```
Comment (ryantordesillas) [inline src/test/java/Bookstore/Model/BookTest.java:44]: You are testing the wrong function lol

**Review by ryantordesillas — CHANGES_REQUESTED** [review]
Looks good! There is just one small error that needs to be fixed but other than that good job

**Comment by ryantordesillas** [comment]
The changes were addressed in #65

**Pull request merged**